english/macgyver/stickfork/stickforkinwall.md

```diff
@@ -2,7 +2,13 @@ You take the fork with both hands and raise it up above your head, dismissing th
 You then plunge the fork with all your might into the wall.  With a muffled whumpf the fork wedges in the crack between
  two marshmallows.   
 
-The marshmellow walls convulse and heave with pain, but eventually settle to their original state.
+The marshmallow walls convulse and heave with pain, but eventually settle to their original state.
 
 Now that the task is done, you stand back and marvel at your handiwork... If there was any sun in the room you could 
 probably use this to tell the time, or you could hang your coat on the newly fashioned coat hook.
+
+Macguyver's work, however, is never finished. As you admire your own marshmallow forking skills, you hear the footsteps of many beings rapidly approaching from behind. You turn around, only to be met with an army of Bill Pullman clones! As they close in, your options are limited.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? This will make the raw file easier to view on the command line, and tools like `git diff` will work work better.